lib/services/assessment.js

```diff
@@ -357,6 +357,10 @@ module.exports = class AssessmentService extends Schmervice.Service {
 
     async patchStudentDetails(key, details={}, txn=null) {
         const { Student } = this.server.models();
+
+        //update type of test
+        details.typeOfTest = "onlineTest";
```

Comment:
patchStudentDetails is also used in google sheet. So, if I change the student partner or student actual stage the function would be called.
So putting it here would be a good idea. Can we move it to somewhere else which is more related to online test specific?

Reply:
I am not getting what are you trying to say that